.adr/0010-interest-bearing-channel.md

```diff
@@ -0,0 +1,130 @@
+# 0010 - Interest Bearing Channel Application
```

Comment:
Could I ask you to:

* increase the counter to **12** (we had another couple of ADRs merged while this has been in preparation)
* cross reference in this docs and in ADR 0010 that this one supercedes that one?

🙏